gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
Read up about .gitignore and ignore files like this from commits. If you plan on changing the .gitignore file, you'll need to do this to not track ignored files: https://stackoverflow.com/a/32377642/4361042